internal/condition/eval/eval.go

```diff
@@ -27,18 +34,30 @@ var conditionEvaluationCostHistogram = promauto.NewHistogram(prometheus.Histogra
 
 // EvaluateTupleCondition returns a bool indicating if the provided tupleKey's condition (if any) was met.
 func EvaluateTupleCondition(
+	ctx context.Context,
 	tupleKey *openfgav1.TupleKey,
 	typesys *typesystem.TypeSystem,
 	context *structpb.Struct,
 ) (*condition.EvaluationResult, error) {
+	ctx, span := tracer.Start(ctx, "EvaluateTupleCondition")
+	defer span.End()
+
+	span.SetAttributes(attribute.String("tuple_key", tupleKey.String()))
+
 	tupleCondition := tupleKey.GetCondition()
 	conditionName := tupleCondition.GetName()
 	if conditionName != "" {
+		span.SetAttributes(attribute.String("condition_name", conditionName))
```

Comment:
This one is arguably redundant since the `tuple_key` attribute already logs the condition name

Reply:
I debated that. I will remove it in favor of the tuple_key.